content/en/docs/how-tos/testing-operator-sdk-operators.md

```diff
@@ -49,7 +49,7 @@ images:
   to: "tested-operator"
 operator:
   bundles: # entries create bundle images from Dockerfiles and an index containing all bundles
-  - as: my-bundle                         # optional
+  - as: my-bundle
```

Comment:
Why is this no longer optional?

Reply:
That is the name of the tag of the built bundle image.
The workflow refers to this name.

It was optional because the bundle was consumed by workflow via index image whose name is `ci-index` for unnamed bundles.